podcasts/TranscriptModel.swift

```diff
@@ -52,7 +52,10 @@ struct TranscriptModel: Sendable {
         var cues = [TranscriptCue]()
         for cue in subtitles.cues {
             let text = cue.text
-            let attributedText = NSAttributedString(string: text + "\n")
+
+            let filteredText: String = ComposeFilter.transcriptFilter.filter(text)
```

Comment:
In the future ww may want to consider using different filters depending of the subtitle file format.